src/libtriton/includes/pythonBindings.hpp

```diff
@@ -14,6 +14,8 @@
   #include <python2.7/Python.h>
   #include <python2.7/longintrepr.h>
 #elif _WIN32
+  #include <cmath>
+  #define _hypot hypot
```

Comment:
Ok. Can you open a new issue for this mingw error and do a PR with theae both lines?